symfony/phpunit-bridge/3.3/.env.test

```diff
@@ -1,6 +1,4 @@
 # define your env variables for the test env here
-KERNEL_CLASS=App\\Kernel
+KERNEL_CLASS='App\Kernel'
 APP_SECRET='s$cretf0rt3st'
 SHELL_VERBOSITY=-1
-SYMFONY_DEPRECATIONS_HELPER=999999
```

Comment:
Why not keep this here? This would solve the problem, that bin/phpunit sets the SYMFONY_DEPRECATIONS_HELPER variable while vendor/bin/phpunit does not set it. This was my major pain point in https://github.com/symfony/recipes/pull/442#issuecomment-407994227

Reply:
I'm not sure how that's related: defining this var in phpunit.xml.dist does work AFAIK, *except* from the "disabled" value (which has to be set in `bin/phpunit`. The reason is another comment from @weaverryan: loading earlier creates an inconsistency in terms of precedence we'd prefer avoiding.

Reply:
I don't see how ryans comment disagrees with my proposal. Keep this line here instead of bin/phpunit.

This way `bin/phpunit` and `vendor/bin/phpunit` behaves the same:
- both set the APP_ENV to test
- both load the boostrap file
- both load .env.test file
- both behave the same in regard to SYMFONY_DEPRECATIONS_HELPER becaues it's part of .env.test

Reply:
OK, got it, updated.